### DISCUSSION TITLE
help to setup webcam

### DISCUSSION THREAD

**Discussion by ride62** [SuRGeoNix/Flyleaf#532] [Q&A]
I'm trying to make a webcam viewer/capture. 
I managed to open the stream using:
player.Open("device://dshow?video=webcamname");
but I don't know how to add the parameters to configure the device (fps, video_size, video_codec)

**Comment by SuRGeoNix** [comment]
Hi @ride62, check this https://www.ffmpeg.org/ffmpeg-devices.html#Options-5

and just add the parameters like this device://dshow?video=webcamname&optX=valueX

**Comment by ride62** [comment]
it worked, thank you so much.
